docs/csharp/language-reference/operators/patterns.md

```diff
@@ -70,9 +71,9 @@ Beginning with C# 9.0, for that purpose you can use a *type pattern*, as the fol
 
 Like a declaration pattern, a type pattern matches an expression when an expression result is non-null and its run-time type satisfies any of the conditions listed above.
 
-You can also use this pattern for a clear, concise `null` check:
+To check for non-null, you can use a [negated](#logical-patterns) `null` [constant pattern](#constant-pattern), as the following example shows:
```

Comment:
nit-picking here: `not null` is neither a declaration nor type pattern

Reply:
You're correct, it is a negated constant pattern.